OpenChange/RTFHandler.m

```diff
@@ -771,19 +764,7 @@ - (NSMutableData *) parse
             }
           else if (*(_bytes+1) == '*')
             {
-	      int cc = 1;
-	      
-	      do
-                {
-                  if (*_bytes == '{')
-                    cc++;
-                  if (*_bytes == '}')
-                    cc--;
-                  
-                  ADVANCE;
-                }
-              while (cc != 0);
-
+              [self parseIgnoringEverything];
               continue;
```

Comment:
This continue is important! We have just ignore the whole `{\*\foo{\whatever....}... }`